### DISCUSSION TITLE
Airdrop and Gnosis

### DISCUSSION THREAD

**Discussion by jvel07** [hashlips-lab/nft-erc721-collection#437] [General]
Hi, @liarco! Hope you are doing good! :)

Please, I got a couple of questions regarding the repo:
- Is there any function for doing “airdrops”?
- I was reading that having the funds with Gnosis is a good practice, would it be needed to edit some code to achieve this?

Thanks so much in advance!

**Comment by liarco** [comment]
Hi @jvel07,
1. the `mintForAddress()` function can be used by the owner for things such as airdrops
2. this definitely requires some dedicated tests to make sure you handle it in a proper way, but you can safely send your contract balance to a GnosisSafe wallet instead of the contract owner

There is much more to say about these topics, if you wanna know more please consider joining the [Open Devs Crew](https://opendevs.io) 😉
